node-module-system/calculate.js

```diff
@@ -0,0 +1,23 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+switch (process.argv[3]) {
+  case 'plus':
+  case '+':
+    console.log(add(process.argv[2], process.argv[4]));
```

Comment:
You use `process.argv[2]` and `process.argv[4]` repeatedly. It would be nice to use it just once.